opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -13,8 +13,149 @@
+
+        Args:
+            instrument_type: This is an instrument matching attribute: the
+                class the instrument must be to match the view.
+
+            instrument_name: This is an instrument matching attribute: the name
+                the instrument must have to match the view. Wild card
+                characters are supported.
+
+            meter_name: This is an instrument matching attribute: the name
+                the instrument meter must have to match the view. Wild card
+                characters are supported.
+
+            meter_version : This is an instrument matching attribute: the
+                version the instrument meter must have to match the view.
+                Wild card characters are supported.
+
+            meter_schema URL : This is an instrument matching attribute: the
+                schema URL the instrument meter must have to match the view.
+
+            name: This is a metric stream customizing attribute: the name of
+                the metric stream. If `None`, the name of the instrument will
+                be used.
+
+            description: This is a metric stream customizing attribute: the
+                description of the metric stream. If `None`, the description of
+                the instrument will be used.
+
+            attribute_keys: This is a metric stream customizing attribute: this
+                is a set of attribute keys. If not `None` then only the
+                measurement attributes that are in `attribute_keys` will be
+                used to identify the metric stream.
+
+            aggregation: This is a metric stream customizing attribute: the
+                aggregatation instance to use when data is aggregated for the
+                corresponding metrics stream. If `None` the default aggregation
+                of the instrument will be used.
+
+        This class is not intended to be subclassed by the user.
+        """
+
+        if (
+            instrument_type
+            is instrument_name
+            is meter_name
+            is meter_version
+            is meter_schema_url
+            is None
+        ):
+            raise Exception(
+                "Some instrument selection "
+                f"criteria must be provided for View {name}"
+            )
+
+        self._name = name
+        self._instrument_type = instrument_type
+        self._instrument_name = instrument_name
+        self._meter_name = meter_name
+        self._meter_version = meter_version
+        self._meter_schema_url = meter_schema_url
+
+        self._description = description
+        self._attribute_keys = attribute_keys
+        self._aggregation = aggregation
+
+        if self._name is not None:
+            self._previously_matched_instrumentation_infos = set()
+
+    # pylint: disable=too-many-return-statements
+    # pylint: disable=too-many-branches
+    @final
+    def _match(self, instrument: Instrument) -> bool:
```

Comment:
This should be `match` and now `_match`. Either that or there should be a separate `match` method that calls `_match`

Reply:
Sorry, what should be the name of this method? `match` or `_match`?

Reply:
the caller expects `match`

Reply:
Refactored to only use `_match`.